### PR TITLE
model_tools: read dataset classes

### PR DESCRIPTION
**Pull request by as-suvorov**

### CODE REVIEW

File: tools/model_tools/src/openvino/model_zoo/omz_data_downloader.py [hunk 1]
```diff
@@ -30,6 +30,12 @@ def copy_data(output_dir):
         str(data_path),
     )
 
+
+def read_dataset_classes(file_name):
+    file_path = _common.PACKAGE_DIR / 'data' / 'dataset_classes' / file_name
```
Comment (eaidova) [inline tools/model_tools/src/openvino/model_zoo/omz_data_downloader.py:35]: do we need to check that such file exists?

File: tools/model_tools/src/openvino/model_zoo/omz_data_downloader.py [hunk 2]
```diff
@@ -30,6 +30,12 @@ def copy_data(output_dir):
...
+    return file_path.read_text()
```
Comment (eaidova) [inline tools/model_tools/src/openvino/model_zoo/omz_data_downloader.py:36]: as I can see, for some datasets, classes represented in json format, probably we need to handle file type during reading?

Which expected result from reading? just get text and parse it outside of tool or retrun list with class names?

File: tools/model_tools/src/openvino/model_zoo/omz_data_downloader.py (hunk 1 above)
Comment (as-suvorov) [reply tools/model_tools/src/openvino/model_zoo/omz_data_downloader.py:36]: check added

File: tools/model_tools/src/openvino/model_zoo/omz_data_downloader.py (hunk 2 above)
Comment (as-suvorov) [reply tools/model_tools/src/openvino/model_zoo/omz_data_downloader.py]: parsing added

**Review by eaidova — APPROVED** [review]

**Pull request merged**